stackdriver-core/docs/toc.json

```diff
@@ -0,0 +1,28 @@
+{
+  "guides": [
+    {
+      "title": "FAQ",
+      "id": "faq",
+      "edit": "https://github.com/GoogleCloudPlatform/gcloud-common/edit/master/faq/readme.md",
+      "contents": "https://raw.githubusercontent.com/GoogleCloudPlatform/gcloud-common/master/faq/readme.md"
+    },
+    {
+      "title": "Troubleshooting",
+      "id": "troubleshooting",
+      "edit": "https://github.com/GoogleCloudPlatform/gcloud-common/edit/master/troubleshooting/readme.md",
+      "contents": "https://raw.githubusercontent.com/GoogleCloudPlatform/gcloud-common/master/troubleshooting/readme.md"
+    },
+    {
+      "title": "Contributing",
+      "id": "contributing",
+      "edit": "https://github.com/GoogleCloudPlatform/gcloud-common/edit/master/contributing/readme.md",
+      "contents": "https://raw.githubusercontent.com/GoogleCloudPlatform/gcloud-common/master/contributing/readme.md"
+    }
+  ],
+  "services": [
+    {
+      "title": "Stackdriver Core",
+      "type": "stackdriver/core"
+    }
```

Comment:
Do you want to add stackdriver/core/tracecontext to this jsondoc menu? Since that's the significant part of this gem?

Also, I think there should an entry in [manifest.json](https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/6279ddc2f792be8824d021b87c8b45f659ba024e/docs/manifest.json), so it shows up on the jsondoc gems list.